--- taca/jclicauthor.desktop	2020-04-09 19:33:04.706691363 +0200
+++ jclicauthor.desktop	2020-04-09 19:55:54.868595272 +0200
@@ -2,7 +2,11 @@
 Version=1.0
 Type=Application
 Name=JClic Author
-GenericName=JClic authoring tool
+Name[es]=JClic Author
+Name[ca]=JClic Author
+GenericName=Educational activities authoring tool
+GenericName[es]=Herramienta de creación de actividades educativas
+GenericName[ca]=Eina de creació d'activitats educatives
 Comment=Tool to create and modify JClic projects
 Comment[ca]=Eina per crear i modificar projectes JClic
 Comment[es]=Herramienta para crear y modificar proyectos JClic
@@ -23,11 +26,13 @@
 Comment[ast]=Aplicación pa xugar a les xeres de JClic.
 Comment[bs]=Aplikacije za stvaranje i uređivanje JClic aktivnosti.
 Comment[vec]=Programa par prodùxer e modifegar atività de JClic
-Keywords=JClic;game;education;
+Keywords=JClic;game;education;school;multimedia;
 Exec=jclicauthor %U
 TryExec=jclicauthor
-Icon=jclicauthor
+Icon=edu.xtec.JClic.jclicauthor
 Terminal=false
 Categories=Education;
 StartupNotify=true
 MimeType=application/jclic;
+X-Desktop-File-Install-Version=0.24
+X-Flatpak=edu.xtec.JClic
